src/components/card/card.jsx

```diff
@@ -1,9 +1,38 @@
-function Card() {
-    return ( 
+
+
+function Card({product}) {
+
+    const { imageSource , name  ,discountedPrice , originalPrice } = product
+
```

Comment:
You can destructure them directly above too like - 
```js
function Card ({ imageSource , name  ,discountedPrice , originalPrice }){
return ()
}
```

Reply:
I passed the whole obj as props so i destructured product from that  .